### PR TITLE
Pricing tweaks

### PR DESCRIPTION
**Pull request by devinaconley**
- define constant addresses explicitly as `Address` type
- helper method for directly pricing GYSR token

### CODE REVIEW

File: config/kovan.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
   "usd_native_address": "0x0000000000000000000000000000000000000000",
   "usd_native_address_v3": "0x0000000000000000000000000000000000000000",
   "usd_weth_address": "0x0000000000000000000000000000000000000000",
+  "gysr_native_address": "0x0000000000000000000000000000000000000000",
```
Comment (IainM22) [inline config/kovan.json:17]: Don't need to bother on this PR but we can consider removing the kovan configurations/scripts

**Review by IainM22 — APPROVED** [review]
Nice

File: config/kovan.json (hunk 1 above)
Comment (devinaconley) [reply config/kovan.json:17]: it's already removed in another branch, i'll do some cleanup/merge after this goes in!

**Comment by devinaconley** [comment]
thanks @IainM22 :ok_hand:

**Pull request merged**